client/src/ui/molecules/OrcidDetails.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { TextField, TwoColumnLayout } from '../atoms';
+
+export interface AuthorDetails {
+    authorFirstName: string;
+    authorLastName: string;
+    authorEmail: string;
+    institution: string;
+}
+
+interface Props {
+    getOrcidDetails(): AuthorDetails;
+}
+
+const OrcidDetails = ({ getOrcidDetails }: Props, authorDetails: AuthorDetails): JSX.Element => {
+    const [authorFirstName, setAuthorFirstName] = useState(authorDetails.authorFirstName || '');
+    const [authorLastName, setAuthorLastName] = useState(authorDetails.authorLastName || '');
+    const [authorEmail, setAuthorEmail] = useState(authorDetails.authorEmail || '');
+    const [institution, setInstitution] = useState(authorDetails.institution || '');
+
+    const getDetails = (): void => {
+        const {
+            authorFirstName: newAuthorFirstName,
+            authorLastName: newAuthorLastName,
+            authorEmail: newAuthorEmail,
+            institution: newInstitution,
+        } = getOrcidDetails();
+        setAuthorFirstName(newAuthorFirstName);
+        setAuthorLastName(newAuthorLastName);
+        setAuthorEmail(newAuthorEmail);
+        setInstitution(newInstitution);
+    };
+
+    return (
+        <div>
+            <div className="orcid-details__link_text">
+                <span onClick={getDetails} className="typography__body typography__body--link">
```

Comment:
Prefil should probably be wrapped in a toggle prop. This component might be worth abstracting a little to make it and AuthorFieldSet component with `prefilToggle`. That way it could be used by someone without ORCID to get the same info.